docs/usage.rst

```diff
@@ -25,6 +25,11 @@ And extract data from those elements::
     >>> selector.xpath('//h1/text()').getall()
     ['Hello, Parsel!']
 
+Extract inner text
+
+    >>> selector.css('h1').get(text=True)
+    'Hello, Parsel!'
+
```

Comment:
I don’t think we need this added here. The goal of this part of the usage documentation is not to cover everything, but to show some minimal Parsel code to illustrate how it works, and I don’t think we need to feature inner text extraction here (since it does not help with the simple input here).

Reply:
Alright